superset-frontend/src/theme/index.ts

```diff
@@ -64,6 +64,11 @@ const baseConfig: ThemeConfig = {
       borderRadiusSM: supersetTheme.gridUnit / 2,
       defaultBg: supersetTheme.colors.grayscale.light4,
     },
+    Progress: {
+      fontSize: supersetTheme.typography.sizes.s,
```

Comment:
@geido Is it possible to specify these at the component?

Reply:
This was a decision we made to keep the style centralized rather than scattered across. The component tokens are relevant to general theme, as the general theme might need to be tweaked if we realize that multiple components are using the same styles and thus should be promoted. We can reconsider this choice once the big part of the transition is done and the general theme should be stable.